src/appfl/comm/mpi/mpi_server_communicator.py

```diff
@@ -170,22 +189,16 @@ def _update_global_model(
                 == "SyncScheduler"
             ), "Batched clients are only supported with SyncScheduler."
             self.logger.info(
-                f"Received UpdateGlobalModel request from batched clients: {client_ids}"
+                f"Received UpdateGlobalModel request from batched clients: {client_ids} [MPI rank {client_rank}]"
```

Comment:
Are we keeping the UpdateGlobalModel terminology here?

Reply:
Github UI wraps/hides some lines of codes. This line is defined in `_update_global_model` function, so it is right to use `UpdateGlobalModel`.